pyrefly/lib/alt/types/class_metadata.rs

```diff
@@ -331,11 +336,20 @@ impl Display for ClassSynthesizedFields {
 /// A struct representing a class's metaclass. A value of `None` indicates
 /// no explicit metaclass, in which case the default metaclass is `type`.
 #[derive(Clone, Debug, TypeEq, PartialEq, Eq, Default)]
-struct Metaclass(Option<ClassType>);
+struct Metaclass {
+    value: Option<ClassType>,
+    explicit: bool,
```

Comment:
i believe this change is not necessary if you check extends_abc. For future reference (not relevant ot this PR)`custom_metaclass` on the class metadata will only return something if the metaclass is not the default.